openmdao/docs/openmdao_book/tests/test_jupyter_output_linting.py

```diff
@@ -59,7 +59,8 @@ def test_output(self):
         for file in _get_files():
             with self.subTest(file):
                 with open(file) as f:
-                    json_data = json.load(f)
+                    data = f.read()
+                    json_data = json.loads(data)
```

Comment:
Just curious, why load the file into a string then convert string to json vs. just loading json from the file?

Reply:
Just an oversight on my part, will fix.